WordPress/Classes/System/SplitViewRootPresenter.swift

```diff
@@ -129,7 +129,11 @@ final class SplitViewRootPresenter: RootViewPresenter {
             }
         )
         let navigationVC = UINavigationController(rootViewController: sitePickerVC)
-        navigationVC.modalPresentationStyle = .formSheet
+        navigationVC.modalPresentationStyle = .popover
+        navigationVC.popoverPresentationController?.sourceView = splitVC.view
```

Comment:
There is probably no difference in practice, but I'm wondering is the `sourceRect` in split view or side bar view? [The `.global` coordinate space's documentation](https://developer.apple.com/documentation/swiftui/coordinatespace/global) says "the root view", which I think is the side bar view controller's view?

Reply:
I understand it as the top-most view controller, which is `splitVC` in our case – it is gets displayed as a root view controller (I hope). It takes the full screen space anyway.